clients/client-shell/cmds/d2g/d2g.go

```diff
@@ -29,6 +29,7 @@ To convert a task definition (JSON), you must use the task definition flag (-t,
   cat /path/to/input/task-definition.json | taskcluster d2g -t
   echo '{"image": "ubuntu", "command": ["bash", "-c", "echo hello world"], "maxRunTime": 300}' | taskcluster d2g`,
 	}
+	cmd.Flags().StringP("engine", "e", "docker", "The container engine to use between docker and podman.")
```

Comment:
i think there was a readme about this somewhere - do we need to update to include information about the -e option?